pkgs/by-name/ni/nibtools/package.nix

```
@@ -0,0 +1,47 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, opencbm
+, cc65
+}:
+
+stdenv.mkDerivation {
+  pname = "nibtools";
+  version = "unstable-2024-02-22";
+
+  src = fetchFromGitHub {
+    owner = "OpenCBM";
+    repo = "nibtools";
+    rev = "1d69f64eec55031c346a2ce115227479f9579a8a";
+    hash = "sha256-+hnkj0uevURkRboTH8WbSt82pZTdWL4ii2PKr6NO0Cg=";
+  };
+
+  NIX_CFLAGS_COMPILE = "-Wno-error=format-security";
+
+  nativeBuildInputs = [
+    cc65
+  ];
+
+  buildInputs = [
+    opencbm
+  ];
+
+  buildPhase = ''
+    mkdir build
+    make -f GNU/Makefile linux
+  '';
```

```suggestion
  preBuild = "mkdir build";
  makeFlags = [ "-f" "GNU/Makefile" "linux" ];
```

it's best to not override phases if you don't have to.